language/diem-framework/modules/DiemAccount.move

```diff
@@ -1427,6 +1428,12 @@ module DiemAccount {
 
     native fun create_signer(addr: address): signer;
 
+    public fun publish_crsn(account: &signer, size: u64)
+    acquires DiemAccount {
+        let account_state = borrow_global<DiemAccount>(Signer::address_of(account));
+        CRSN::publish(account, account_state.sequence_number, size)
```

Comment:
It would be useful to add a comment here justifying why this should *not* be `account_state.sequence_number + 1` just to reassure the reader (it took me a second to convince myself :) ).